### ISSUE TITLE
Could not load the Qt platform plugin "windows" in "" even though it was found.

### ISSUE THREAD

**Issue by meg-bit** [napari/napari#3261]
## 🐛 Bug

In windows 10, I cannot use napari in python. The error message is 
"Could not load the Qt platform plugin "windows" in "" even though it was found."

**_When I run it in jupyter notebook, the kernel always dies._**

## To Reproduce
import napari
viewer = napari.view_image(img)

**Comment by tlambert03** [comment]
Can you please tell us how you created your environment, how you installed napari, and the output of "conda list" (if in conda, or "pip list" if not)?

Thanks

**Comment by meg-bit** [comment]
conda create -y -n napari-env python=3.8
conda activate napari-env
pip install 'napari[all]'

**Comment by meg-bit** [comment]
I can open napari by clicking its icon in the start menu. If I open the IPython console inside napari and run 
viewer = napari.view_image(img), I get the following error:

```
Jupyter QtConsole 5.1.0
Python 3.8.3 (tags/v3.8.3:6f8c832, May 13 2020, 22:37:02) [MSC v.1924 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 7.24.1 -- An enhanced Interactive Python. Type '?' for help.

viewer = napari.view_image('./data/20210714registered_tiff/amp-ch1-cycle1.tiff')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-e8c9236fb616> in <module>
----> 1 viewer = napari.view_image('./data/20210714registered_tiff/amp-ch1-cycle1.tiff')

~\AppData\Local\Programs\napari\\app\napari\view_layers.py in view_image(*args, **kwargs)
    132 @_merge_layer_viewer_sigs_docs
    133 def view_image(*args, **kwargs):
--> 134     return _make_viewer_then('add_image', args, kwargs)
    135 
    136 

~\AppData\Local\Programs\napari\\app\napari\view_layers.py in _make_viewer_then(add_method, args, kwargs)
    117         kwargs.update(kwargs.pop("kwargs"))
    118     method = getattr(viewer, add_method)
--> 119     method(*args, **kwargs)
    120     return viewer
    121 

~\AppData\Local\Programs\napari\\app\napari\components\viewer_model.py in add_image(self, data, channel_axis, rgb, colormap, contrast_limits, gamma, interpolation, rendering, iso_threshold, attenuation, name, metadata, scale, translate, rotate, shear, affine, opacity, blending, visible, multiscale)
    692                         )
    693                     )
--> 694             layer = Image(data, **kwargs)
    695             self.layers.append(layer)
    696 

~\AppData\Local\Programs\napari\\app\napari\layers\image\image.py in __init__(self, data, rgb, colormap, contrast_limits, gamma, interpolation, rendering, iso_threshold, attenuation, name, metadata, scale, translate, rotate, shear, affine, opacity, blending, visible, multiscale)
    201         # Determine if data is a multiscale
    202         if multiscale is None:
--> 203             multiscale, data = guess_multiscale(data)
    204 
    205         # Determine initial shape

~\AppData\Local\Programs\napari\\app\napari\layers\image\_image_utils.py in guess_multiscale(data)
     54         return False, data
     55 
---> 56     shapes = [d.shape for d in data]
     57     sizes = np.array([np.prod(shape, dtype=np.uint64) for shape in shapes])
     58     if len(sizes) == 1 and (isinstance(data, list) or isinstance(data, tuple)):

~\AppData\Local\Programs\napari\\app\napari\layers\image\_image_utils.py in <listcomp>(.0)
     54         return False, data
     55 
---> 56     shapes = [d.shape for d in data]
     57     sizes = np.array([np.prod(shape, dtype=np.uint64) for shape in shapes])
     58     if len(sizes) == 1 and (isinstance(data, list) or isinstance(data, tuple)):

AttributeError: 'str' object has no attribute 'shape'
```

**Comment by tlambert03** [comment]
ok, I'm a little confused by some of the things I'm seeing here.  in your installation log in https://github.com/napari/napari/issues/3261#issuecomment-906877419, it's saying "Requirement already satisfied: ... in **c:\users\whh\anaconda3\lib\site-packages**"  which does suggest that it's not installing into from your newly created `napari-env` ... Instead, it seems you're always using your global base environment (which makes it very difficult to debug).  I'd still like to see the output of `conda list` (not the installation log).  This is a tough one to debug. you might have a messed up root environment which can make things rather difficult (might want to reinstall anaconda)

> I can open napari by clicking its icon in the start menu.

This part and the fact that it's loading from `~\AppData\Local\Programs\napari\app` suggests that you're using the bundled app, and not the one you installed with pip.  does that sound correct?  In any case, the argument you're providing to `view_image` isn't quite right:

`napari.view_image(...)` is looking for a _numpy array_ not a string path.
If you'd like to open a file from a filepath, please use [`napari.view_path`](https://napari.org/api/stable/napari.html#napari.view_path)

**Comment by meg-bit** [comment]
I tried the napari-env and reinstalling PyQt5 and PySide2, the error is the same as if I go from the base. I eventually used the bundled app but the parameters I can set are limited, so I start to debug napari for python again. Let me send you the conda list.
```
(napari-env) PS C:\Users\whh> conda list
# packages in environment at C:\Users\whh\anaconda3\envs\napari-env:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                   pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
attrs                     21.2.0                   pypi_0    pypi
babel                     2.9.1                    pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
blas                      1.0                         mkl
blosc                     1.21.0               h19a0ad4_0
brotli                    1.0.9                ha925a31_2
bzip2                     1.0.8                he774522_0
ca-certificates           2021.7.5             haa95532_1
cachey                    0.2.1                    pypi_0    pypi
certifi                   2021.5.30        py38haa95532_0
cfitsio                   3.470                he774522_6
charls                    2.2.0                h6c2663c_0
charset-normalizer        2.0.4                    pypi_0    pypi
cloudpickle               1.6.0                      py_0
colorama                  0.4.4                    pypi_0    pypi
cycler                    0.10.0                   py38_0
cytoolz                   0.11.0           py38he774522_0
daal4py                   2021.3.0         py38h757b272_0
dal                       2021.3.0           haa95532_564
dask-core                 2021.8.0           pyhd3eb1b0_0
debugpy                   1.4.1                    pypi_0    pypi
decorator                 5.0.9                    pypi_0    pypi
docstring-parser          0.10                     pypi_0    pypi
docutils                  0.17.1                   pypi_0    pypi
fonttools                 4.25.0             pyhd3eb1b0_0
freetype                  2.10.4               hd328e21_0
freetype-py               2.2.0                    pypi_0    pypi
fsspec                    2021.7.0           pyhd3eb1b0_0
giflib                    5.2.1                h62dcd97_0
heapdict                  1.0.1                    pypi_0    pypi
hsluv                     5.0.2                    pypi_0    pypi
icc_rt                    2019.0.0             h0cc432a_1
icu                       58.2                 ha925a31_3
idna                      3.2                      pypi_0    pypi
imagecodecs               2021.6.8         py38h5da4933_0
imageio                   2.9.0              pyhd3eb1b0_0
imagesize                 1.2.0                    pypi_0    pypi
intel-openmp              2021.3.0          haa95532_3372
ipykernel                 6.2.0                    pypi_0    pypi
ipython                   7.26.0                   pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
jedi                      0.18.0                   pypi_0    pypi
jinja2                    3.0.1                    pypi_0    pypi
joblib                    1.0.1              pyhd3eb1b0_0
jpeg                      9b                   hb83a4c4_2
jsonschema                3.2.0                    pypi_0    pypi
jupyter-client            6.1.12                   pypi_0    pypi
jupyter-core              4.7.1                    pypi_0    pypi
kiwisolver                1.3.1            py38hd77b12b_0
lcms2                     2.12                 h83e58a3_0
lerc                      2.2.1                hd77b12b_0
libaec                    1.0.4                h33f27b4_1
libdeflate                1.7                  h2bbff1b_5
libpng                    1.6.37               h2a8f88b_0
libtiff                   4.2.0                hd0e1b90_0
libzopfli                 1.0.3                ha925a31_0
locket                    0.2.1            py38haa95532_1
lz4-c                     1.9.3                h2bbff1b_1
magicgui                  0.2.10                   pypi_0    pypi
markupsafe                2.0.1                    pypi_0    pypi
matplotlib                3.4.2            py38haa95532_0
matplotlib-base           3.4.2            py38h49ac443_0
matplotlib-inline         0.1.2                    pypi_0    pypi
mkl                       2021.3.0           haa95532_524
mkl-service               2.4.0            py38h2bbff1b_0
mkl_fft                   1.3.0            py38h277e83a_2
mkl_random                1.2.2            py38hf11a4ad_0
munkres                   1.1.4                      py_0
napari                    0.4.10                   pypi_0    pypi
napari-console            0.0.3                    pypi_0    pypi
napari-plugin-engine      0.1.9                    pypi_0    pypi
napari-svg                0.1.5                    pypi_0    pypi
networkx                  2.6.2              pyhd3eb1b0_0
numpy                     1.20.3           py38ha4e8547_0
numpy-base                1.20.3           py38hc2deb75_0
numpydoc                  1.1.0                    pypi_0    pypi
olefile                   0.46                       py_0
openjpeg                  2.3.0                h5ec785f_1
openssl                   1.1.1k               h2bbff1b_0
packaging                 21.0               pyhd3eb1b0_0
pandas                    1.2.4            py38hd77b12b_0
parso                     0.8.2                    pypi_0    pypi
partd                     1.2.0              pyhd3eb1b0_0
patsy                     0.5.1                    py38_0
pickleshare               0.7.5                    pypi_0    pypi
pillow                    8.3.1            py38h4fa10fc_0
pint                      0.17                     pypi_0    pypi
pip                       21.0.1           py38haa95532_0
pooch                     1.4.0                    pypi_0    pypi
prompt-toolkit            3.0.19                   pypi_0    pypi
psutil                    5.8.0                    pypi_0    pypi
pydantic                  1.8.2                    pypi_0    pypi
pygments                  2.10.0                   pypi_0    pypi
pyopengl                  3.1.5                    pypi_0    pypi
pyparsing                 2.4.7              pyhd3eb1b0_0
pyqt                      5.9.2            py38ha925a31_4
pyqt5                     5.15.4                   pypi_0    pypi
pyqt5-qt5                 5.15.2                   pypi_0    pypi
pyqt5-sip                 12.9.0                   pypi_0    pypi
pyrsistent                0.18.0                   pypi_0    pypi
python                    3.8.11               h6244533_1
python-dateutil           2.8.2              pyhd3eb1b0_0
pytz                      2021.1             pyhd3eb1b0_0
pywavelets                1.1.1            py38he774522_2
pywin32                   301                      pypi_0    pypi
pyyaml                    5.4.1            py38h2bbff1b_1
pyzmq                     22.2.1                   pypi_0    pypi
qt                        5.9.7            vc14h73c81de_0
qtconsole                 5.1.1                    pypi_0    pypi
qtpy                      1.10.0                   pypi_0    pypi
requests                  2.26.0                   pypi_0    pypi
scikit-image              0.18.1           py38hf11a4ad_0
scikit-learn              0.24.2           py38hf11a4ad_1
scikit-learn-intelex      2021.3.0         py38haa95532_0
scipy                     1.6.2            py38h66253e8_1
seaborn                   0.11.2             pyhd3eb1b0_0
setuptools                52.0.0           py38haa95532_0
sip                       4.19.13          py38ha925a31_0
six                       1.16.0             pyhd3eb1b0_0
snappy                    1.1.8                h33f27b4_0
snowballstemmer           2.1.0                    pypi_0    pypi
sphinx                    4.1.2                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.2                    pypi_0    pypi
sphinxcontrib-devhelp     1.0.2                    pypi_0    pypi
sphinxcontrib-htmlhelp    2.0.0                    pypi_0    pypi
sphinxcontrib-jsmath      1.0.1                    pypi_0    pypi
sphinxcontrib-qthelp      1.0.3                    pypi_0    pypi
sphinxcontrib-serializinghtml 1.1.5                    pypi_0    pypi
sqlite                    3.36.0               h2bbff1b_0
statsmodels               0.12.2           py38h2bbff1b_0
superqt                   0.2.2                    pypi_0    pypi
tbb                       2021.3.0             h59b6b97_0
threadpoolctl             2.2.0              pyhbf3da8f_0
tifffile                  2021.7.2           pyhd3eb1b0_2
tk                        8.6.10               he774522_0
toolz                     0.11.1             pyhd3eb1b0_0
tornado                   6.1              py38h2bbff1b_0
tqdm                      4.62.1                   pypi_0    pypi
traitlets                 5.0.5                    pypi_0    pypi
typing-extensions         3.10.0.0                 pypi_0    pypi
tzdata                    2021a                h5d7bf9c_0
urllib3                   1.26.6                   pypi_0    pypi
vc                        14.2                 h21ff451_1
vispy                     0.7.3                    pypi_0    pypi
vs2015_runtime            14.27.29016          h5e58377_2
wcwidth                   0.2.5                    pypi_0    pypi
wheel                     0.37.0             pyhd3eb1b0_0
wincertstore              0.2                      py38_0
wrapt                     1.12.1                   pypi_0    pypi
xz                        5.2.5                h62dcd97_0
yaml                      0.2.5                he774522_0
zfp                       0.5.5                hd77b12b_6
zlib                      1.2.11               h62dcd97_4
zstd                      1.4.9                h19a0ad4_0
```

**Comment by sofroniewn** [comment]
@jaimergp do you have any ideas here?

**Comment by tlambert03** [comment]
hi @meg-bit,
you can see in your conda list output that you've got pyqt5.9 in your environment installed by conda 

```
pyqt 5.9.2 py38ha925a31_4
qt 5.9.7 vc14h73c81de_0
```
_in addition_ to the qt installed by pip:

```
pyqt5 5.15.4 pypi_0 pypi
pyqt5-qt5 5.15.2 pypi_0 pypi
pyqt5-sip 12.9.0 pypi_0 pypi
```

this is very likely to be the source of your problems.  It seems likely that you also installed something into this environment, and that perhaps you don't have conda forge in your channels.  note that the pyqt being shipped from the (default) anaconda is still pyqt5.9 (https://anaconda.org/anaconda/pyqt) ... so, for instance, if you do something like `conda install notebook` either before or after installing napari, _and_ if you don't have conda forge in your channels, then you could easily end up with this situation.

please see https://github.com/napari/napari/issues/2710#issuecomment-840694404 and [this thread](https://forum.image.sc/t/napari-issues-on-bigsur/52630/10?u=talley) for more details.

if you did do additional installations (like notebook) into this environment, and did _not_ use the conda forge channel, please try again, without installing other things into the environment, run `napari` on the command line.  If it works, then you can add note book using `conda install -c conda-forge jupyter notebook`

**Comment by tlambert03** [comment]
side not @napari/core-devs ... 
we should add this caveat to the install info.  It's common enough that people a) run `conda install jupyter notebook` into the napari environment after installing napari and b) don't have `conda-forge` in their channels...

**Comment by meg-bit** [comment]
Thank you! I will try this to see it resolves the issue.

**Comment by tlambert03** [comment]
hi @meg-bit ... just checking in.  did you get this figured out?

**Comment by meg-bit** [comment]
Sorry, I switched to working on something else since this did not work and have not tried to reinstall it. I will close the issue after I try and if it works. Thanks for following up.

**Comment by meg-bit** [comment]
I uninstalled everything and created a new environment for it. Before I installed jupyter notebook using `conda install -c conda-forge jupyter notebook`, **I first ran napari in the command line to check. Napari does not run even without installing jupyter notebook yet.** 

(napari-env) PS C:\Users\whh> napari
Traceback (most recent call last):
  File "c:\users\whh\anaconda3\envs\napari-env\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\whh\anaconda3\envs\napari-env\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\whh\anaconda3\envs\napari-env\Scripts\napari.exe\__main__.py", line 7, in <module>
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\__main__.py", line 420, in main
    _run()
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\__main__.py", line 230, in _run
    from napari import run, view_path
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\_lazy.py", line 47, in __getattr__
    submod = import_module(f'{module_name}.{attr_to_modules[name]}')
  File "c:\users\whh\anaconda3\envs\napari-env\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\view_layers.py", line 17, in <module>
    from .viewer import Viewer
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\viewer.py", line 3, in <module>
    from .components import ViewerModel
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\components\__init__.py", line 19, in <module>
    from .layerlist import LayerList
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\components\layerlist.py", line 8, in <module>
    from ..layers import Layer
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\layers\__init__.py", line 10, in <module>
    from .base import Layer
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\layers\base\__init__.py", line 1, in <module>
    from .base import Layer
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\layers\base\base.py", line 9, in <module>
    from ...utils.dask_utils import configure_dask
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\utils\__init__.py", line 2, in <module>
    from .dask_utils import resize_dask_cache
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\napari\utils\dask_utils.py", line 8, in <module>
    import dask
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\dask\__init__.py", line 3, in <module>
    from .base import annotate, compute, is_dask_collection, optimize, persist, visualize
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\dask\base.py", line 17, in <module>
    from tlz import curry, groupby, identity, merge
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\tlz\_build_tlz.py", line 54, in exec_module
    toolz_mods = self._load_toolz(module.__name__)
  File "c:\users\whh\anaconda3\envs\napari-env\lib\site-packages\tlz\_build_tlz.py", line 28, in _load_toolz
    raise ImportError(fullname)
ImportError: tlz.curry

+++++++++++++++++++++++++++++++++++++
conda list

(napari-env) PS C:\Users\whh> conda list
\# packages in environment at C:\Users\whh\anaconda3\envs\napari-env:
\#
\# Name                    Version                   Build  Channel
alabaster                 0.7.12                   pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
attrs                     21.2.0                   pypi_0    pypi
babel                     2.9.1                    pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
ca-certificates           2021.7.5             haa95532_1
cachey                    0.2.1                    pypi_0    pypi
certifi                   2021.5.30        py38haa95532_0
charset-normalizer        2.0.4                    pypi_0    pypi
cloudpickle               1.6.0                    pypi_0    pypi
colorama                  0.4.4                    pypi_0    pypi
cycler                    0.10.0                   pypi_0    pypi
dask                      2021.8.1                 pypi_0    pypi
debugpy                   1.4.1                    pypi_0    pypi
decorator                 5.0.9                    pypi_0    pypi
docstring-parser          0.10                     pypi_0    pypi
docutils                  0.17.1                   pypi_0    pypi
entrypoints               0.3                      pypi_0    pypi
freetype-py               2.2.0                    pypi_0    pypi
fsspec                    2021.8.1                 pypi_0    pypi
heapdict                  1.0.1                    pypi_0    pypi
hsluv                     5.0.2                    pypi_0    pypi
idna                      3.2                      pypi_0    pypi
imageio                   2.9.0                    pypi_0    pypi
imagesize                 1.2.0                    pypi_0    pypi
ipykernel                 6.3.1                    pypi_0    pypi
ipython                   7.27.0                   pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
jedi                      0.18.0                   pypi_0    pypi
jinja2                    3.0.1                    pypi_0    pypi
jsonschema                3.2.0                    pypi_0    pypi
jupyter-client            7.0.2                    pypi_0    pypi
jupyter-core              4.7.1                    pypi_0    pypi
kiwisolver                1.3.2                    pypi_0    pypi
locket                    0.2.1                    pypi_0    pypi
magicgui                  0.2.10                   pypi_0    pypi
markupsafe                2.0.1                    pypi_0    pypi
matplotlib                3.4.3                    pypi_0    pypi
matplotlib-inline         0.1.2                    pypi_0    pypi
napari                    0.4.10                   pypi_0    pypi
napari-console            0.0.4                    pypi_0    pypi
napari-plugin-engine      0.1.9                    pypi_0    pypi
napari-svg                0.1.5                    pypi_0    pypi
nest-asyncio              1.5.1                    pypi_0    pypi
networkx                  2.6.2                    pypi_0    pypi
numpy                     1.21.2                   pypi_0    pypi
numpydoc                  1.1.0                    pypi_0    pypi
openssl                   1.1.1l               h2bbff1b_0
packaging                 21.0                     pypi_0    pypi
parso                     0.8.2                    pypi_0    pypi
partd                     1.2.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    8.3.2                    pypi_0    pypi
pint                      0.17                     pypi_0    pypi
pip                       21.0.1           py38haa95532_0
pooch                     1.5.1                    pypi_0    pypi
prompt-toolkit            3.0.20                   pypi_0    pypi
psutil                    5.8.0                    pypi_0    pypi
pydantic                  1.8.2                    pypi_0    pypi
pygments                  2.10.0                   pypi_0    pypi
pyopengl                  3.1.5                    pypi_0    pypi
pyparsing                 2.4.7                    pypi_0    pypi
pyqt5-qt5                 5.15.2                   pypi_0    pypi
pyqt5-sip                 12.9.0                   pypi_0    pypi
pyrsistent                0.18.0                   pypi_0    pypi
python                    3.8.11               h6244533_1
python-dateutil           2.8.2                    pypi_0    pypi
pytz                      2021.1                   pypi_0    pypi
pywavelets                1.1.1                    pypi_0    pypi
pywin32                   301                      pypi_0    pypi
pyyaml                    5.4.1                    pypi_0    pypi
pyzmq                     22.2.1                   pypi_0    pypi
qtconsole                 5.1.1                    pypi_0    pypi
qtpy                      1.10.0                   pypi_0    pypi
requests                  2.26.0                   pypi_0    pypi
scikit-image              0.18.3                   pypi_0    pypi
scipy                     1.7.1                    pypi_0    pypi
setuptools                52.0.0           py38haa95532_0
six                       1.16.0                   pypi_0    pypi
snowballstemmer           2.1.0                    pypi_0    pypi
sphinx                    4.1.2                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.2                    pypi_0    pypi
sphinxcontrib-devhelp     1.0.2                    pypi_0    pypi
sphinxcontrib-htmlhelp    2.0.0                    pypi_0    pypi
sphinxcontrib-jsmath      1.0.1                    pypi_0    pypi
sphinxcontrib-qthelp      1.0.3                    pypi_0    pypi
sphinxcontrib-serializinghtml 1.1.5                    pypi_0    pypi
sqlite                    3.36.0               h2bbff1b_0
tifffile                  2021.8.30                pypi_0    pypi
toolz                     0.11.1                   pypi_0    pypi
tornado                   6.1                      pypi_0    pypi
tqdm                      4.62.2                   pypi_0    pypi
traitlets                 5.1.0                    pypi_0    pypi
typing-extensions         3.10.0.2                 pypi_0    pypi
urllib3                   1.26.6                   pypi_0    pypi
vc                        14.2                 h21ff451_1
vispy                     0.8.1                    pypi_0    pypi
vs2015_runtime            14.27.29016          h5e58377_2
wcwidth                   0.2.5                    pypi_0    pypi
wheel                     0.37.0             pyhd3eb1b0_0
wincertstore              0.2                      py38_0
wrapt                     1.12.1                   pypi_0    pypi

**Comment by jaimergp** [comment]
Hm, `toolz` is listed there and there have not been any recent releases. Might be a `sys.path` issue? Can you run `python -m napari` instead? Also, let's check for the output of the following commands after activating your `napari-env` environment.

```
python -c "import sys, os; print(sys.executable); print(*sys.path, sep='\n'); print(*sorted(os.environ.items()), sep='\n')"
```

Thanks!

**Comment by meg-bit** [comment]
Thank you. I checked. 

Running `python -m napari` results in the same error as running `napari`.

Running the commands you gave me:

(napari-env) PS C:\Users\whh> python -c "import sys, os; print(sys.executable); print(*sys.path, sep='\n'); print(*sorted(os.environ.items()), sep='\n')"
>>
C:\Users\whh\anaconda3\envs\napari-env\python.exe

C:\Users\whh\anaconda3\envs\napari-env\python38.zip
C:\Users\whh\anaconda3\envs\napari-env\DLLs
C:\Users\whh\anaconda3\envs\napari-env\lib
C:\Users\whh\anaconda3\envs\napari-env
C:\Users\whh\AppData\Roaming\Python\Python38\site-packages
C:\Users\whh\anaconda3\envs\napari-env\lib\site-packages
C:\Users\whh\anaconda3\envs\napari-env\lib\site-packages\win32
C:\Users\whh\anaconda3\envs\napari-env\lib\site-packages\win32\lib
C:\Users\whh\anaconda3\envs\napari-env\lib\site-packages\Pythonwin
('ALLUSERSPROFILE', 'C:\\ProgramData')
('APPDATA', 'C:\\Users\\whh\\AppData\\Roaming')
('COMMONPROGRAMFILES', 'C:\\Program Files\\Common Files')
('COMMONPROGRAMFILES(X86)', 'C:\\Program Files (x86)\\Common Files')
('COMMONPROGRAMW6432', 'C:\\Program Files\\Common Files')
('COMPUTERNAME', 'DESKTOP-FFB306P')
('COMSPEC', 'C:\\Windows\\system32\\cmd.exe')
('CONDA_DEFAULT_ENV', 'napari-env')
('CONDA_EXE', 'C:\\Users\\whh\\anaconda3\\Scripts\\conda.exe')
('CONDA_PREFIX', 'C:\\Users\\whh\\anaconda3\\envs\\napari-env')
('CONDA_PREFIX_1', 'C:\\Users\\whh\\anaconda3')
('CONDA_PROMPT_MODIFIER', '(napari-env) ')
('CONDA_PYTHON_EXE', 'C:\\Users\\whh\\anaconda3\\python.exe')
('CONDA_SHLVL', '2')
('DRIVERDATA', 'C:\\Windows\\System32\\Drivers\\DriverData')
('FPS_BROWSER_APP_PROFILE_STRING', 'Internet Explorer')
('FPS_BROWSER_USER_PROFILE_STRING', 'Default')
('HOMEDRIVE', 'C:')
('HOMEPATH', '\\Users\\whh')
('LOCALAPPDATA', 'C:\\Users\\whh\\AppData\\Local')
('LOGONSERVER', '\\\\DESKTOP-FFB306P')
('NUMBER_OF_PROCESSORS', '8')
('ONEDRIVE', 'C:\\Users\\whh\\OneDrive')
('OS', 'Windows_NT')
('PATH', 'C:\\Users\\whh\\anaconda3\\envs\\napari-env;C:\\Users\\whh\\anaconda3\\envs\\napari-env\\Library\\mingw-w64\\bin;C:\\Users\\whh\\anaconda3\\envs\\napari-env\\Library\\usr\\bin;C:\\Users\\whh\\anaconda3\\envs\\napari-env\\Library\\bin;C:\\Users\\whh\\anaconda3\\envs\\napari-env\\Scripts;C:\\Users\\whh\\anaconda3\\envs\\napari-env\\bin;C:\\Users\\whh\\anaconda3\\condabin;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0;C:\\Windows\\System32\\OpenSSH;C:\\Program Files\\MATLAB\\R2021a\\bin;C:\\Program Files\\Git\\cmd;C:\\Users\\whh\\AppData\\Local\\Microsoft\\WindowsApps;.')
('PATHEXT', '.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL')
('PROCESSOR_ARCHITECTURE', 'AMD64')
('PROCESSOR_IDENTIFIER', 'Intel64 Family 6 Model 140 Stepping 1, GenuineIntel')
('PROCESSOR_LEVEL', '6')
('PROCESSOR_REVISION', '8c01')
('PROGRAMDATA', 'C:\\ProgramData')
('PROGRAMFILES', 'C:\\Program Files')
('PROGRAMFILES(X86)', 'C:\\Program Files (x86)')
('PROGRAMW6432', 'C:\\Program Files')
('PSEXECUTIONPOLICYPREFERENCE', 'Bypass')
('PSMODULEPATH', 'C:\\Users\\whh\\Documents\\WindowsPowerShell\\Modules;C:\\Program Files (x86)\\WindowsPowerShell\\Modules;C:\\Program Files\\WindowsPowerShell\\Modules;C:\\Windows\\system32\\WindowsPowerShell\\v1.0\\Modules;C:\\Program Files (x86)\\Google\\Cloud SDK\\google-cloud-sdk\\platform\\PowerShell')
('PUBLIC', 'C:\\Users\\Public')
('SESSIONNAME', 'Console')
('SYSTEMDRIVE', 'C:')
('SYSTEMROOT', 'C:\\Windows')
('TEMP', 'C:\\Users\\whh\\AppData\\Local\\Temp')
('TMP', 'C:\\Users\\whh\\AppData\\Local\\Temp')
('USERDOMAIN', 'DESKTOP-FFB306P')
('USERDOMAIN_ROAMINGPROFILE', 'DESKTOP-FFB306P')
('USERNAME', 'whh')
('USERPROFILE', 'C:\\Users\\whh')
('WINDIR', 'C:\\Windows')
('ZES_ENABLE_SYSMAN', '1')
('_CONDA_EXE', 'C:/Users/whh/anaconda3\\Scripts\\conda.exe')
('_CONDA_ROOT', 'C:/Users/whh/anaconda3')

**Comment by tlambert03** [comment]
interesting, looks like you have a global python install site-packages in front of your environment's site-packages in your path `C:\Users\whh\AppData\Roaming\Python\Python38\site-packages`

@jaimergp ... do you agree that might be problematic?

**Comment by jaimergp** [comment]
Absolutely. That path there is probably added by another installation and is messing with the import mechanisms. It might have been set through the registry or a .pth file. To make sure you are not using it, you can temporarily rename that directory to something else (don't delete just yet, it might be important for other programs in your system), and let's see if that solves it!

TLDR:

Rename `C:\Users\whh\AppData\Roaming\Python` to `C:\Users\whh\AppData\Roaming\Python-bak` and let's see what happens.
